packages/block-editor/src/components/off-canvas-editor/style.scss

```diff
@@ -1,397 +1,15 @@
-.block-editor-list-view-tree {
-	width: 100%;
-	border-collapse: collapse;
+.offcanvas-editor__appender .block-editor-inserter__toggle {
+	background-color: #1e1e1e;
+	color: #fff;
+	margin: $grid-unit-10 0 0 28px;
+	border-radius: 2px;
+	height: 24px;
+	min-width: 24px;
 	padding: 0;
-	margin: 0;
 
-	// Move upwards when in modal.
-	.components-modal__content & {
-		margin: (-$grid-unit-15) (-$grid-unit-15 * 0.5) 0;
-		width: calc(100% + #{ $grid-unit-15 });
-	}
-}
-
-.block-editor-list-view-leaf {
```

Comment:
Where did all this CS go? Do we not need it?

Reply:
AFAIK all that was unused until I included the file in the main CSS includes file. So yes as it's just duplicating.

Reply:
So this change is correct then?